planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/utils/calculation.cpp

```diff
@@ -42,4 +44,47 @@ double calc_dist_from_pose_to_terminal_end(
   }
   return utils::getDistanceToEndOfLane(src_pose, lanes);
 }
+
+double calc_dist_to_last_fit_width(
+  const lanelet::ConstLanelets lanelets, const Pose & src_pose,
+  const BehaviorPathPlannerParameters & common_param, const double margin)
+{
+  if (lanelets.empty()) return 0.0;
+
+  const auto lane_polygon = lanelets.back().polygon2d().basicPolygon();
+  const auto center_line = lanelet::utils::generateFineCenterline(lanelets.back(), 1.0);
+  universe_utils::LineString2d line_string;
+  line_string.reserve(center_line.size() - 1);
```

Comment:
are we sure that `center_line`  is not empty ? If we are not sure please add a check to avoid doing `0U - 1`.